EScavange.py

```diff
@@ -0,0 +1,18 @@
+import database
+import gamemaker
+
+class main():
```

Comment:
I thought we were going with EScavenge for the name here. Also, class names are capitalized.